spec/features/admin/order_cycles_spec.rb

```diff
@@ -332,11 +332,15 @@
       select 'My supplier', from: 'new_supplier_id'
       click_button 'Add supplier'
       expect(page).to have_selector("table.exchanges tr.supplier", text: "My supplier")
-      page.all("table.exchanges tr.supplier td.products").each(&:click)
 
+      page.all("table.exchanges tr.supplier td.products").each do |tab|
+        tab.click
+        sleep 1
```

Comment:
is this spec not unrelated to the PR?
anyway, if this is a good change, we need a very good reason to use a sleep in a spec.
otherwise we can use expect on an element you are waiting for and capybara will wait for it.

Reply:
Ah right, I think I unintentionally committed this, strangely the tests were failing around this, the introduction of this sleep actually helped the test pass, I just wanted ensure that my changes were not the cause, I will un-commit this now